index.html

```diff
@@ -23,16 +23,18 @@
     <h1 id="welcome">Welcome!</h1>
     <!-- TODO: Replace this tagline with something less narcissistic -->
     <h3>We are one of the most successful and inventive FIRST Robotics teams in the Chesapeake District.</h3>
-    <a href="/robot/2018" class="button">View our 2018 Robot</a>
+    <a href="/robot/2017" class="button">View our 2017 Robot</a>
```

Comment:
Yeah this stuff is because of the duplicates.

Reply:
oof